d2l-icon-button-shared-styles.html

```diff
@@ -20,9 +20,7 @@
 					outline-style: none;
 				};
 				--d2l-icon-button-icon: {
-					height: var(--d2l-icon-height);
 					transition: color 0.3s ease;
-					width: var(--d2l-icon-width);
```

Comment:
Not needed since `d2l-icon` handles its own width/height based on the icon being used. Plus if `--d2l-icon-height` (or width) weren't set, the value ending up in the CSS was blank and invalid.